widgets/modules/EnsEMBL/Web/Component/Transcript/AFDB.pm

```diff
@@ -0,0 +1,63 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2017] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package EnsEMBL::Web::Component::Transcript::AFDB;
+
+use strict;
+
+use HTML::Entities qw(encode_entities);
+use URI::Escape;
+
+use base qw(EnsEMBL::Web::Component::Transcript EnsEMBL::Web::Component::AFDB);
+
+sub _init {
+  my $self = shift;
+  $self->cacheable(0);
+  $self->ajaxable(1);  
+}
+
+sub content {
+  my $self        = shift;
+
+  my $hub         = $self->hub;
+  my $object      = $self->object;
+  my $html;
+  if ($object->Obj->isa('Bio::EnsEMBL::Transcript')) {
```

Comment:
i dont think you need this check. You are already in a transcript component.

Reply:
Ok. I believe this is mostly just copy-pasting from the PDBE module.

Reply:
Deleted the check.